doc/themes/scikit-learn-modern/nav.html

```diff
@@ -7,15 +7,16 @@
 {%- endif %}
 
 {%- set drop_down_navigation = [
-  ('Getting Started', 'getting_started'),
-  ('Tutorial', 'tutorial/index'),
-  ('Glossary', 'glossary'),
-  ('Development', 'developers/index'),
-  ('FAQ', 'faq'),
-  ('Related packages', 'related_projects'),
-  ('Roadmap', 'roadmap'),
-  ('About us', 'about'),
-  ('Documentation', 'documentation')]
+  ('Getting Started', pathto('getting_started')),
+  ('Tutorial', pathto('tutorial/index')),
+  ('Glossary', pathto('glossary')),
+  ('Development', pathto('developers/index')),
+  ('FAQ', pathto('faq')),
+  ('Related packages', pathto('related_projects')),
+  ('Roadmap', pathto('roadmap')),
+  ('About us', pathto('about')),
+  ('Other Versions', 'https://scikit-learn.org/dev/versions.html'),
+  ('Documentation', pathto('documentation'))]
```

Comment:
I now find this placement quite awkward

Reply:
Are you referring to the documentation link?

Reply:
Yes, awkward to appear under Other Versions. 

Maybe if it's called "documentation home" it's a bit clearer what its role is